datauri/datauri.py

```diff
@@ -33,6 +45,21 @@ def __init__(self, media_type, data, uri):
         self.data = data
         self.uri = uri
 
+    @CachedProperty
+    def charset(self):
+        prefix = 'charset='
+        chunks = self.media_type.split(';')
+        for chunk in chunks:
+            if chunk.startswith(prefix):
+                return chunk[len(prefix):]
+        return None
+
+    @CachedProperty
+    def text(self):
+        if not self.media_type.startswith('text/'):
```

Comment:
intuitively i'd say utf-8 is a saner and more useful default. or does the spec explicitly forbid that?

Reply:
Yes, it is:
> If one is not specified, the media type of the data URI is assumed to be text/plain;charset=US-ASCIIQ

Reply:
ok cool ascii it is then